tests/org.eclipse.passage.lic.base.tests/src/org/eclipse/passage/lic/internal/base/tests/conditions/evaluation/BasePermissionEmittingServiceTest.java

```diff
@@ -171,7 +189,7 @@ private Condition corrupted() {
 				.withValidityPeriod(hour());
 	}
 
-	private Condition humanoid(int legs, int eyes) {
+	private FakeCondition humanoid(int legs, int eyes) {
```

Comment:
Is there a reason to narrowing down return type here?

Reply:
Yes, otherwise the method `withValidityPeriod()` cannot be called on the returned object in the two new test cases.